tools/cmake/logging.cmake

```diff
@@ -1,7 +1,3 @@
 # Configuration for logging.
 set( LOGGING_INCLUDE_DIRS
-     ${ROOT_DIR}/platform/include )
-
-set( LOGGING_SOURCES
-     ${ROOT_DIR}/platform/posix/logging.c
-     ${ROOT_DIR}/platform/posix/clock_posix.c )
\ No newline at end of file
+     ${ROOT_DIR}/platform/include )
```

Comment:
Why remove these?

Reply:
Will add source files in a follow-up PR that will replace use of `printf` with a logging function